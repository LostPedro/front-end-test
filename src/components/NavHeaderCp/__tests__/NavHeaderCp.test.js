import React from 'react'
import {render, cleanup} from '@testing-library/react'
import {NavHeaderCp} from '../'

afterEach(cleanup)

it('should take a snapshot', () => {
  const {asFragment} = render(<NavHeaderCp />)

  expect(asFragment(<NavHeaderCp />)).toMatchSnapshot()
})
