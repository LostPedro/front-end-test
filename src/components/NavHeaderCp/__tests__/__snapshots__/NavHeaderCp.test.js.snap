// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="nav-header-component"
  >
    <div
      aria-hidden="true"
      class="nav-header-component-nav-icon-wrapper"
      role="button"
    >
      <div />
    </div>
    <span
      class="nav-header-component-nav-header-title"
    />
    <div
      aria-hidden="true"
      class="nav-header-component-nav-icon-wrapper"
      role="button"
    >
      <div />
    </div>
  </div>
</DocumentFragment>
`;
